doc/rados/operations/crush-map.rst

```diff
@@ -3,103 +3,103 @@
 ============
 
 The :abbr:`CRUSH (Controlled Replication Under Scalable Hashing)` algorithm
-determines how to store and retrieve data by computing storage locations.
-CRUSH empowers Ceph clients to communicate with OSDs directly rather than
-through a centralized server or broker. With an algorithmically determined
+computes storage locations in order to determine how to store and retrieve
+data.  CRUSH allows Ceph clients to communicate with OSDs directly rather than
+through a centralized server or broker. By using an algorithmically-determined
 method of storing and retrieving data, Ceph avoids a single point of failure, a
 performance bottleneck, and a physical limit to its scalability.
 
-CRUSH uses a map of your cluster (the CRUSH map) to pseudo-randomly
-map data to OSDs, distributing it across the cluster according to configured
-replication policy and failure domain.  For a detailed discussion of CRUSH, see
+CRUSH uses a map of the cluster (the CRUSH map) to pseudorandomly map data to
```

Comment:
I'm not sure that `pseudorandomly` adds meaning here, I would consider dropping it.

Reply:
Accepted.